pkg/util/fs.go

```diff
@@ -40,6 +42,15 @@ type fs struct {
 	runner CommandRunner
 }
 
+// JoinPath joins the path elements always using the UNIX way. This function
+// should be used when constructing path that is valid for running in container.
+func JoinPath(elem ...string) string {
+	if runtime.GOOS == "windows" {
+		return path.Join(elem...)
+	}
+	return filepath.Join(elem...)
```

Comment:
@soltysh yeah this function is pointless, I can just use path.Join on places where it is required... I'm reworking this PR anyway :-)